docs/source/tutorials/benchmarking/bm_simulator.rst

```diff
@@ -162,6 +162,11 @@ This call runs a number of experiments sequentially on the local machine:
   where ``A = n_workers``, ``B = benchmark.n_workers``. This means we run for
   longer if ``n_workers < benchmark.n_workers``, but keep
   ``benchmark.max_wallclock_time`` the same otherwise.
+* ``use_long_tuner_name_prefix``: If 1, results for an experiment are written
+  to a directory whose prefix is
+  :code:`f"{experiment_tag}-{benchmark_name}-{seed}"`, followed by a postfix
```

Comment:
Are seeds limited to three characters? 

Also, how does one set the experiment tag?

Reply:
No, why? Seeds are numbers starting from 0.

The `Tuner` writes results (and checkpoints) to a directory whose name has prefix `tuner_path`, and it appends a datetime plus a 3-digit hash, so things are always unique.

This PR is about choosing what `tuner_name` should be. Currently, it is just `experiment_tag`, which is typically a short string you choose. I run comparative experiments where 1000 or even 10000 experiments share the same `experiment_tag`, they differ by method, seed, benchmark, etc.

To tell these apart, you need to read the `metadata.json` files for each of them and filter according to values for seed and benchmark in there.

So, a longer `tuner_name` is redundant, because the info is in the metadata, but it simplifies basic things like re-run parts of a large experiment.

Reply:
Thanks for the clarification, makes sense!